packages/odyssey-storybook/src/components/odyssey-mui/Card/Card.stories.tsx

```diff
@@ -199,3 +199,18 @@ export const ButtonWithoutImage: StoryObj<typeof Card> = {
     </Box>
   ),
 };
+
+export const ClickableMenu: StoryObj<CardProps> = {
+  render: ({ ...props }) => (
+    <Box sx={{ maxWidth: 262 }}>
+      <Card
+        {...props}
+        overline={props.overline}
+        title={props.title}
+        description={props.description}
+        onClickMenuButton={() => alert("Clicked on menu")}
+        onClick={() => alert("Clicked on Card")}
+      />
+    </Box>
+  ),
+};
```

Comment:
Instead of using `...props` here, it'd be better to be explicit about what props we're passing.

Using `{...props}` makes it hard to see what props are being passed into `Card`.